apps/polls/assets/Polls.jsx

```diff
@@ -6,18 +6,9 @@ var Poll = React.createClass({
   getInitialState: function () {
     // FIXME: example data
     return {
-      title: 'Getrennte Eltern: Ist das Wechselmodell die beste Lösung für alle?',
-      options: [{
-        label: 'Ja',
-        count: 22434
-      }, {
-        label: 'Nein',
-        count: 40062
-      }, {
-        label: 'Vielleicht',
-        count: 17627
-      }],
-      ownVote: null,
+      title: this.props.poll.title,
```

Comment:
As we discussed, see https://shripadk.github.io/react/tips/initial-ajax.html. We will keep this pattern for consistency now though.